nur/update.py

```diff
@@ -184,6 +188,7 @@ def eval_repo(spec: RepoSpec, repo_path: Path) -> None:
             "-I", f"nixpkgs={nixpkgs_path()}",
             "-I", str(repo_path),
             "-I", str(eval_path),
+            "-I", str("/home/infinisil/src/NUR/evalRepo.nix")
```

Comment:
If everything would be in `lib` then `"-I", str(ROOT.joinpath("lib"))` would cover it.